src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -205,7 +206,7 @@ public class RunResult
     {
         public bool Failed { get; set; }
 
-        public Exception Exception { get; set; }
+        public ExceptionDispatchInfo Exception { get; set; }
```

Comment:
Would it make sense to change the name of this property since the type changed as well?